static/js/lib/util.js

```diff
@@ -111,3 +111,11 @@ export const makeUUID = (len: number) =>
     .map(int => int.toString(16))
     .join("")
     .slice(0, len)
+
+export const removeTrailingSlash = (str: string) =>
+  R.compose(
+    R.equals("/"),
+    R.last
+  )(str)
+    ? R.dropLast(1, str)
+    : str
```

Comment:
could this just be `str.replace(/\/$/, "")`?

Reply:
Yeah, I tried to get too cute with this. I generally try to avoid regexes in really simple use cases like this, but I did a quick performance test and this is definitely worse than the regex equivalent. I also tested a dirt-simple solution using `substr` and that performed twice as well as the regex, so I went with that.

Reply:
sure, my concern is more code readability than performance for a trivial function like this, but I think the new version is fine